app/useUser.tsx

```diff
@@ -61,8 +52,19 @@ export const UserContextProvider = ({ children }: { children: ReactChild }) => {
     };
   }, []);
 
+  useEffect(() => {
```

Comment:
Is it good to separate these in two useEffects?

Reply:
Doesn't really matter one way or the other, maybe should put them back into one, I was testing something and didn't revert this code